src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -126,6 +126,13 @@
      */
     private String appLabel;
 
+    /**
+     * A short description about the application
+     *
+     * Default: Empty string.
+     */
+    private String appDescription;
```

Comment:
These two don't belong to Android. If they are Win specific, add a comment `// Windows'  before them. If they are valid for all platforms, put them before the Mac ones (I see they are).

Reply:
Currently these are windows specific, but I can foresee these being used in other platforms as well.